templates/components/select.build.hbs

```diff
@@ -2,7 +2,7 @@
   <select class="form-control hidden-select" :name="name" :id="name" v-model="value" :required="required" v-on:change="updateValue()">
     <option v-if="placeholder" value="">\{{ placeholder }}</option>
     <option v-for="option in options" :value="option.id" :disabled="option.disabled">
-      \{{ option.id }}
+      \{{ option.label }}
```

Comment:
what about existing forms that in their settings have only the `id`? We need to support those too for existing apps even if they don't go through the process of opening the form interface and saving changes. Does using a fallback work here? e.g. `{{ option.label || option.id }}`

Reply:
Are there any cases how I can test this behavior with my changes?

Reply:
1. setup a form locally before making your changes
2. switch the branch to include your changes
3. refresh the output iframe (http://localhost:3000/build) and see if works

You can also test out in a similar way on staging:

1. publish your form component without your changes
2. drop it into a screen and set it up as required
3. publish your form component with the changes (under the same package name)
4. refresh the screen (without making any change via the interface) and verify if it works

Reply:
Fallback works. Have updated commit